addons/sys_gsa/config.cpp

```diff
@@ -0,0 +1,19 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"acre_main", "acre_sys_core"};
```

Comment:
`acre_sys_core` requires `acre_main` already. I don't know why we do that in other places either though.